corehq/blobs/pillow.py

```diff
@@ -19,46 +14,28 @@
 
 class BlobDeletionProcessor(PillowProcessor):
 
-    def __init__(self, blob_db, bucket_base):
+    def __init__(self, blob_db):
         super(BlobDeletionProcessor, self).__init__()
         self.blob_db = blob_db
-        self.bucket_base = bucket_base
 
     def process_change(self, pillow_instance, change):
         if change.deleted:
-            bucket = join(self.bucket_base, change.id)
+            bucket = "{}/{}".format(change.meta.data_source_name, change.id)
```

Comment:
sorry I think it should be `change.metadata`.

here's [where it get's set](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/change_feed/consumer/feed.py#L146)
and I think the document store definitely has the right DB name or all sorts of stuff wouldn't work